ds_judgements_public_ui/templates/judgment/results.html

```diff
@@ -14,11 +14,28 @@ <h1>{{ pagetitle }}</h1>
     {% include "includes/results_search_component.html" %}
     <div class="results__container">
       <div class="results__list-layout-container">
-        <div class="results__result-controls-container">{% include "includes/result_controls.html" %}</div>
-        <div class="results__result-list-container">
-          {% include "includes/results_list.html" %}
-          {% include "includes/pagination.html" %}
-        </div>
+        {% if context.search_results %}
+          <div class="results__result-controls-container">{% include "includes/result_controls.html" %}</div>
+          <div class="results__result-list-container">
+            {% include "includes/results_list.html" %}
+            {% include "includes/pagination.html" %}
+          </div>
+        {% else %}
+          <h2>We found no results</h2>
+          <p>
+            If you're unable to find a particular judgment, you may wish to check
+            <a href="{% url 'what_to_expect' %}#section-coverage">whether we cover that court and date</a>.
+          </p>
+          <p>
+            In particular, we do not have:
+            <ul>
+              <li>any court judgments before 2003</li>
+              <li>any crown court, county court or magistrate court judgments,</li>
```

Comment:
Should this be "magistrate's court"?

Reply:
magistrates', apparently.
I've also adjusted that on what-to-expect.

I've also changed the capitalisation based on https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#county-court ; I think the capitalisation on what-to-expect is probably fine because they're talking about 'county courts'.

🏘️ 👨‍🎨  :)